code/mission/missionparse.cpp

```diff
@@ -1015,25 +1015,29 @@ void parse_cutscenes(mission *pm)
 
 		while (!optional_string("#end"))
 		{
-			// this list should correspond to the MOVIE_* #defines
-			scene.type = optional_string_one_of(8,
+			// this list should correspond to the MOVIE_* enums
```

Comment:
The code in missionparse.cpp should be reverted (except that changing "#defines" to "enums" is fine).  The previous code handled the situation adequately and the warning is not necessary -- in fact the warning is wrong because the intent is to fully support "$Post-briefing Cutscene" as a valid string.

Reply:
Actually, my intention is to deprecate it and that's what my code does.